packages/core/src/utils/get-type-name.spec.ts

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable max-classes-per-file */
+import { Command, Event } from '../bus';
+import { getTypeName } from './get-type-name';
+
+class Test {}
+
+class TestEvent extends Event<string> {
+  constructor() {
+    super('TestEvent');
+  }
+}
+
+class TestCommand extends Command<string, void> {
+  constructor() {
+    super('TestCommand');
+  }
+}
```

Comment:
You should avoid such deps in fixtures

```suggestion
import { getTypeName } from './get-type-name';

class Test {}

class SubTest extends Test {
  constructor() {
    super();
  }
}
```